### PR TITLE
Add support for Ember 1.10 beta.3 (real HTMLBars)

### PR DESCRIPTION
**Pull request by mixonic**
Ember.js 1.10 beta.1 did not have a robust Handlebars compat mode, and thus HTMLBars wasn't actually being tested in this suite.

This introduces 1.10-beta.3, which has a compat layer and uses HTMLBars.

In beta 4 or 5, the htmlbars compiler will no longer be shipped with prod builds of Ember (@rwjblue is working on this). Thus although I've added a dummy `no-htmlbars.js` file for now, we will soon need to include a compiler package. I've changes the `handlebarsPath` settings and such to be `compilerPath` in prep for this.

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -23,14 +23,12 @@
...
-  var HANDLEBARS_DEFAULT = "/ember/handlebars-v1.3.0.js";
-
   var EMBER_VERSIONS = [
-    { name: '1.9.1',                     path: '/ember/ember.prod-1.9.1.js',       handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.10.0-beta.1+canary',      path: '/ember/1.10.0-beta.1.canary.js',   handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.9.0-beta.3',              path: '/ember/1.9.0-beta.3.js',           handlebarsPath: '/ember/handlebars-v2.0.0.js'},
-    { name: '1.8.1',                     path: '/ember/1.8.1.js',                  handlebarsPath: HANDLEBARS_DEFAULT},
-    { name: '1.7.1',                     path: "/ember/1.7.1.js",                  handlebarsPath: HANDLEBARS_DEFAULT}
+    { name: '1.9.1',                     path: '/ember/ember.prod-1.9.1.js',       compilerPath: '/ember/handlebars-v2.0.0.js'},
+    {name: '1.10.0-beta.3',              path: "/ember/1.10.0-beta.3.js",          compilerPath: "/ember/no-htmlbars.js" },
```
Comment (rwjblue) [inline app/app.js]: Can you add a leading space before `name`?

Comment (rwjblue) [inline app/app.js]: Also, if you make the `compilerPath` actually be `ember-template-compiler.js` (the one that is distributed via Bower and to S3 along with the build) then this is "future proof" for when I remove the compiler in beta.4 or beta.5.

**Comment by rwjblue** [comment]
Left a couple tiny nit-picks but this looks good.

**Comment by mixonic** [comment]
Cleaned up.

**Comment by rwjblue** [comment]
:heart:

**Comment by eviltrout** [comment]
Awesome, loving all the recent changes guys! :heart:

**Comment by stefanpenner** [comment]
Expect more. :)

**Pull request merged**